.github/workflows/CI.yaml

```diff
@@ -459,6 +459,47 @@ jobs:
             -p gear-runtime-interface
             --features=lazy-pages
 
+  build-macos-x86:
```

Comment:
I think we shouldn't make separate run for macos-x86. We can make nightly run like for windows. It's not critically important for us to test each commit for macos.

Reply:
> We can make nightly run like for windows

It's every master push actually. Same will be for this job but I'm testing right now